lib/sheets/comments.js

```diff
@@ -22,13 +22,19 @@ exports.create = function(options) {
     return utils.post(_.extend(optionsToSend, getOptions), callback);
   };
 
+  var editComment = function(putOptions, callback) {
+    optionsToSend.url = buildUrl(putOptions);
+    return utils.put(_.extend(optionsToSend, putOptions), callback);
+  };
+
   var buildUrl = function(urlOptions) {
     return options.apiUrls.sheets + urlOptions.sheetId + '/comments/' + (urlOptions.commentId || '');
   };
 
   return {
     getComment : getComment,
     deleteComment : deleteComment,
-    addCommentAttachment : addCommentAttachment
+    addCommentAttachment : addCommentAttachment,
+    editComment : editComment,
```

Comment:
Trailing comma - I don't _think_ this would raise a fuss? If the linter doesn't complain, then no worries.

Reply:
The linter doesn't complain, but I'll remove it